src/main/java/bookface/model/book/exceptions/BookOnLoanException.java

```diff
@@ -0,0 +1,10 @@
+package bookface.model.book.exceptions;
+
+/**
+ * Signals that the operation cannot be performed since the {@code Book} is currently loaned out
+ */
+public class BookOnLoanException extends RuntimeException {
```

Comment:
Also as this is a RuntimeException and so is unchecked, we probably don't want to throw this as it will not be caught (we are only catching ParseException)